model/postgres/postgres_server.rb

```diff
@@ -246,6 +247,18 @@ def self.run_query(vm, query)
   def run_query(query)
     self.class.run_query(vm, query)
   end
+
+  def metrics_config
```

Comment:
Given that there is no server specific field, how about moving this to image generation? Or they can be hard-coded where we are using them. Any reason for adding these as configs?

Reply:
`bin/monitor` would rely on some of these, specifically the `additional_labels` and `metrics_dir` fields, so wanted to keep them in once place where it can be referenced from both places